Character.cpp

```diff
@@ -23,9 +24,10 @@ void Character::attack( Character& other )
         std::cout << "make another party member use an item to revive them" << std::endl << std::endl;
         return;
     }
-        
+
     isDefending = false;
     std::cout << getName() << " has attacked " << other.getName() << std::endl;
+    other.hitPoints -= attackDamage;
```

Comment:
remove this.   `other.takeDamage()` does all the work of adjusting hitpoints on the next line,
- [ ] revise.